addons/hr_recruitment_integration_base/models/hr_job.py

```diff
@@ -0,0 +1,54 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields, _
+
+
+class Job(models.Model):
+    _inherit = 'hr.job'
+
+    job_post_count = fields.Integer(
+        compute='_compute_job_post_count', string='Number of Job Posts')
+    job_post_ids = fields.One2many('hr.job.post', 'job_id', string='Job Posts')
+    currency_id = fields.Many2one(
+        'res.currency', related='company_id.currency_id', readonly=True)
+    employment_status_id = fields.Many2one('hr.contract.status', string='Employment Status')
+    salary_min = fields.Monetary('Minimum Salary', currency_field='currency_id')
+    salary_max = fields.Monetary('Maximum Salary', currency_field='currency_id')
+    salary_time_unit = fields.Selection([
+        ('hourly', 'Hour'),
+        ('daily', 'Day'),
+        ('weekly', 'Week'),
+        ('biweekly', 'Bi-Week'),
+        ('monthly', 'Month'),
+        ('yearly', 'Year'),
+        ], string='Salary Time Unit', default='monthly')
+    industry_id = fields.Many2one('res.partner.industry', string='Industry')
+
+    def _compute_job_post_count(self):
+        counts_by_job = dict(
+                self.env['hr.job.post']._read_group(
+                    domain=[
+                        ('job_id', 'in', self.ids),
+                        ('status', '!=', 'deleted'),
+                    ],
+                    groupby=['job_id'],
+                    aggregates=['__count'],
+                )
+            )
+        for job in self:
+            job.job_post_count = counts_by_job.get(job, 0)
+
+    @api.onchange('salary_min', 'salary_max', 'salary_time_unit')
+    def _check_salary(self):
```

Comment:
This is not a constraint
```suggestion
    def _onchange_salary(self):
```